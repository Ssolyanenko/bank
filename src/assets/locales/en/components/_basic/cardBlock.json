{
  "cardBlock": {
    "orderACard": "Order a card"
  }
}
