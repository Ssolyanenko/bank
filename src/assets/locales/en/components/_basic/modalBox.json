{
  "modalBox": {
    "codeExpirationTime": "Code expiration time",
    "seconds": "seconds",
    "didNotGetTheCode": "Didnâ€™t get the code?",
    "sendCodeAgain": "Send code again in"
  }
}
