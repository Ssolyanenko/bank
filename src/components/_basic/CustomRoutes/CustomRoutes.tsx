import React, { FC, PropsWithChildren, ReactElement } from 'react';
import { Navigate, Route, Routes } from 'react-router';

import { RoutingPaths } from 'constants/routingPaths';

export const CustomRoutes: FC<PropsWithChildren> = ({ children }): ReactElement => (
  <Routes>
    {children}
    <Route path="*" element={<Navigate to={`/${RoutingPaths.PAGE_NOT_FOUND}`} />} />
  </Routes>
);
