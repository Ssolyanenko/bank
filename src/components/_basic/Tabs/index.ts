export * from './Tabs';
export * from './TabTitle';
