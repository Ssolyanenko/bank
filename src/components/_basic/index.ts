export * from './PrimaryButton';
export * from './Button';
export * from './Input';
export * from './InputPassword';
export * from './ModalWrapper';
export * from './ModalConfirmation';
export * from './PhoneCodeTimer';
export * from './ModalBox';
export * from './CloseButton';
export * from './BackButton';
export * from './StepperForm';
export * from './StepperLabel';
export * from './ChatButton';
export * from './BanTimer';
export * from './CardTemplate';
export * from './SecondaryButton';
export * from './generic';
export * from './SelectStyled';
export * from './TextFieldStyled';
export * from './InputStyled';
export * from './CardTemplate';
export * from './Modal';
export * from './SecureWrapper';
export * from './InputRange';
export * from './DropDown';
export * from './RadioButtonsGroup';
export * from './FormError';
export * from './Checkbox';
export * from './Switcher';
export * from './EditButton';
export * from './CardBlock';
export * from './Tabs';
export * from './ModalSuccess';
export * from './DropDownWithCheckbox';
export * from './SelectFormikTemplate';
export * from './Loader';
export * from './DatePicker';
export * from './CustomRoutes';
