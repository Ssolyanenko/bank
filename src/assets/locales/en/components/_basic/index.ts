import modalBox from './modalBox.json';
import cardBlock from './cardBlock.json';
import stepperForm from './stepperForm.json';

export const translationsBasic = {
  ...modalBox,
  ...cardBlock,
  ...stepperForm,
};
