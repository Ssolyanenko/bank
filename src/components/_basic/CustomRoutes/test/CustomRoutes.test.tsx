import React from 'react';
import { shallow } from 'enzyme';

import { CustomRoutes } from 'components/_basic/CustomRoutes';

describe('<CustomRoutes />', (): void => {
  test('should make a snapshot of component', (): void => {
    const props = {
      children: <div />,
    };
    const wrapper = shallow(<CustomRoutes {...props} />);

    const component = wrapper.getElement();

    expect(component).toMatchSnapshot();
  });
});
