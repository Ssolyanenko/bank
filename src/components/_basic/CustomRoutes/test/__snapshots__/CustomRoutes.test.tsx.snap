// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomRoutes /> should make a snapshot of component 1`] = `
<Routes>
  <div />
  <Route
    element={
      <Navigate
        to="/page-not-found"
      />
    }
    path="*"
  />
</Routes>
`;
